tests/unit/other.tcl

```diff
@@ -99,10 +99,13 @@ start_server {tags {"other"}} {
                 set _ 1
             } else {
                 check_consistency {aofdump} {
+                    r config set appendonly yes
+                    waitForBgrewriteaof r
```

Comment:
same here.. instead of enabling AOF, and then using BGREWRITEAOF, we can just assert that it was disabled, and then enable it (it'll do the rewrite we wanted)